README.md

```diff
@@ -1,32 +1,403 @@
+
+To drill down more, we'll need to nest another loop inside the current loop. For
+this loop, we'll first capture the object associated with the current key
+(`home` or `away`) and save it to a variable, then iterate through _that_
+object:
+
+```js
+function debugPractice() {
+  const game = gameObject();
+  for (const gameKey in game) {
+    const teamObj = game[gameKey];
+    for (const teamKey in teamObj) {
+      // are you ABSOLUTELY SURE what 'teamKey' is?
+      // use debugger to find out!
+      debugger;
+    }
+  }
+}
+```
+
+Refresh the page and take a look at `teamObj` and `teamKey`. Are they what you
+expected?
+
+At this point, you may find (if you haven't already) that you're not seeing the
+key you expected to see. Recall that objects in JavaScript, unlike arrays, are
+not ordered. What this means is that you may not always see the key you expect
+when you inspect variables containing object keys, such as `gameKey` and
+`teamKey`. Inspect the `teamObj` variable. You'll see that it's an object that
+contains three keys, `teamName`, `colors` and `game`. When you inspect
+`teamKey`, as long as it's one of those three values, the iteration is set up
+properly.
+
+You can continue to drill down into the `game` object using the same basic
+technique:
+
+1. Create a variable to contain the object (or array) you want to access.
+2. Create a `for...in` (or `for...of`) loop to iterate through it.
+3. Use the debugger to make sure that you're seeing what you expect. If not, you
+   can use the console to experiment with your variables until you figure out
+   how to access what you need.
+
+**Challenge:** Add the next step to your `debugPractice()` function to iterate
+through the players for each team, using the steps above. Use the debugger to
+make sure you're accessing the stats for each player in turn for each team.
+
+<details><summary><b>Answer</b></summary>
+    <code>const playerData = teamObj.players;</code><br/>
+    <code>for (const key in playerData) {</code><br/>
+    <code>debugger;</code><br/>
+    <code>}</code>
+</details>
+
+NOTE: If you simply add code to loop through the players inside our existing
+code, the code will probably not function the way you are expecting it to. Be
+sure to step through the inner loop, paying attention to the values of the
+variables (in particular, the `key` variable) until you figure out what's going
+on. Once you've done that, see if you can figure out how to fix it.
+
+<details><summary><b>What's causing the bug</b></summary>
+<p>Launch the debugger and take a look at the value of the <code>teamObj</code> variable. Note that it is an object with three properties: <code>teamName</code>, <code>colors</code>, and <code>players</code>. Because we are iterating through that object in the loop that encloses the loop through the player stats, <strong>the inner loop is being executed three times for each team — once for each property in <code>teamObj</code>!<strong></p>
```

Comment:
Maybe add somewhere in here for them to step through the loop again while now also paying attention the `teamKey` so they can clearly see this happening for themselves.